code/modules/surgery/organs/augments_arms.dm

```diff
@@ -216,6 +223,15 @@
 		var/obj/item/assembly/flash/armimplant/F = locate(/obj/item/assembly/flash/armimplant) in items_list
 		F.I = src
 
+/obj/item/organ/cyberimp/arm/flash/Extend()
+	. = ..()
+	holder.set_light_range(7)
+	holder.set_light_on(TRUE)
```

Comment:
What if the holder already had another light, from another source?
Use `mob_light()`, check for examples of its usage around the code.

Reply:
We spoke about this one earlier, but "holder" in this case is the tool, not the arm or mob. There was code in the basic `cyberimp/arm/Extend()` proc to check if the tool was a flash, and apply a light source to it if so. Since `cyberimp/arm/flash` is the only arm implant to have a flash, I just moved the code out of the parent and into the child.

Why the flash implant has a light source, I have no idea. I assume IC it's actually meant to be a flashlight or something.

Reply:
Also, I did a find/paste replace and `holder` has been replaced with `active_item` to better represent what it actually references.